nightwatch/globals.js

```diff
@@ -1,35 +1,110 @@
-const fs = require('fs');
-const path = require('path');
-
+const http = require('../lib/setup.js');
 const setup = require('../lib/setup.js');
 
 let viteServer;
-const projectConfigFile = path.join(process.cwd(), 'vite.config.js');
-const viteConfigFile = path.join(__dirname, '../vite.config.js');
-
-const hasProjectConfigFile = () => {
-  try {
-    return fs.statSync(projectConfigFile).isFile();
-  } catch (err) {
-    return false;
+const isWorker = process.argv.includes('--test-worker');
+const startViteServer = async (settings = {}) =>{
+  settings.vite_dev_server = Object.assign({
+    start_vite: true,
+    port: 5173
+  }, settings.vite_dev_server || {});
+
+  let vite_port;
+  if (settings.vite_dev_server.start_vite) {
+    viteServer = await setup();
+
+    settings.vite_dev_server.port = vite_port = viteServer.config.server.port;
+  } else {
+    vite_port = settings.vite_dev_server.port; 
+    try {
+      const enabled = await makeViteRequest(vite_port);
+      if (!enabled) {
+        const error = new Error('Missing vite-plugin-nightwatch');
+        const code = `:
+
+    import nightwatchPlugin from 'vite-plugin-nightwatch'
+
+    export default {
+      plugins: [
+        // ... other plugins
+        nightwatchPlugin({
+          componentType: 'vue'
+        })
+      ]
+    };
+    `;
+        error.help = ['Please ensure that "vite-plugin-nightwatch" is loaded in your Vite config file ' + code];
+        throw error;
```

Comment:
It's the same error we are throwing in `index.js`, can you move it to different file maybe utils or separate file for throwing an error